src/mcp_atlassian/server.py

```diff
@@ -347,6 +347,11 @@ async def list_tools() -> list[Tool]:
                                 "description": "Whether to include page metadata such as creation date, last update, version, and labels",
                                 "default": True,
                             },
+                            "markdown": {
+                                "type": "boolean",
+                                "description": "Whether to convert page to markdown or keep it in raw HTML format. Can help reaviling macros (like dates) and other content not visible in markdown",
```

Comment:
```suggestion
                                "description": "Whether to convert page to markdown (true) or keep it in raw HTML format (false). Raw HTML can reveal macros (like dates) not visible in markdown, but CAUTION: using HTML significantly increases token usage in AI responses.",
```
I'd recommend updating the parameter description to include a caution about token usage. These changes would help users understand the tradeoff between seeing the full content with macros versus keeping token usage lower.